src/index.html

```diff
@@ -17,11 +17,445 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/index.css"
     />
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#">
+        <img
+          class="img"
+          src="images/logo.png"
+          alt="MOYO_main_logo"
+        />
+      </a>
```

Comment:
It's a good practice to give meaningful names to your classes. The class 'img' is too generic. Consider renaming it to something more descriptive, for example 'logo-img'.